packages/ocaml-variants/ocaml-variants.4.10.0+musl+static+flambda/opam

```diff
@@ -17,6 +17,7 @@ depends: [
 depexts: [
   ["musl-tools"] {os-family = "debian"}
   ["musl"] {os-distribution = "arch"}
+  ["musl-dev"] {os-distribution = "alpine"}
```

Comment:
Is this really needed on Alpine? musl is the default libc on Alpine, and it compiled before without this depext.

Reply:
musl-dev contains the *.a libc required for the static linking: https://pkgs.alpinelinux.org/contents?branch=edge&name=musl-dev&arch=x86_64&repo=main